cypress/e2e/webTables.cy.js

```diff
@@ -1,7 +1,124 @@
 /// <reference types='cypress' />
+Cypress.on('uncaught:exception', (_err) => {
+  return false;
+});
+
+let user;
 
 describe('Web Tables page', () => {
-  it('', () => {
+  before(() => {
+    // eslint-disable-next-line arrow-parens
+    cy.task('generateUser').then(generateUser => {
+      user = generateUser;
+    });
+  });
+
+  beforeEach(() => {
+    cy.visit('/webtables');
+  });
+
+  it('Should check pagination and selector of rows.', () => {
+    cy.get('.-previous > .-btn').should('contain', 'Previous');
+    cy.get('.-next > .-btn').should('contain', 'Next');
+    cy.get('.-previous > .-btn').should('exist');
+    cy.get('.-next > .-btn').should('exist');
+    cy.get('select').select('5 rows');
+    cy.get('select').should('contain', '5 rows');
+    cy.get('#addNewRecordButton').click();
+    cy.get('#firstName').type(user.firstName);
+    cy.get('#lastName').type(user.lastName);
+    cy.get('#userEmail').type(user.email);
+    cy.get('#age').type(user.age);
+    cy.get('#salary').type(user.salary);
+    cy.get('#department').type(user.department);
+    cy.get('#submit').click();
+    cy.get('#addNewRecordButton').click();
+    cy.get('#firstName').type(user.firstName);
+    cy.get('#lastName').type(user.lastName);
+    cy.get('#userEmail').type(user.email);
+    cy.get('#age').type(user.age);
+    cy.get('#salary').type(user.salary);
+    cy.get('#department').type(user.department);
+    cy.get('#submit').click();
+    cy.get('#addNewRecordButton').click();
+    cy.get('#firstName').type(user.firstName);
+    cy.get('#lastName').type(user.lastName);
+    cy.get('#userEmail').type(user.email);
+    cy.get('#age').type(user.age);
+    cy.get('#salary').type(user.salary);
+    cy.get('#department').type(user.department);
+    cy.get('#submit').click();
+    cy.get('#addNewRecordButton').click();
+    cy.get('#firstName').type(user.firstName);
+    cy.get('#lastName').type(user.lastName);
+    cy.get('#userEmail').type(user.email);
+    cy.get('#age').type(user.age);
+    cy.get('#salary').type(user.salary);
+    cy.get('#department').type(user.department);
+    cy.get('#submit').click();
+    cy.contains('Next').click();
```

Comment:
```suggestion
    cy.contains('Next').click();
```
again, you didn't assert that your click has changed something. When you click the button you need to check that you are on the 2nd page. Then you should click the [previous] button and check that page is 1

Reply:
Changes in commit https://github.com/mate-academy/qa_cypress_e2e_web_tables/pull/188/commits/a38dbac156617e5fbc07bf1a11770c0082a88050